src/paginator.js

```diff
@@ -0,0 +1,300 @@
+import assign from 'lodash.assign';
+import omit from 'lodash.omit';
+
+
+export const styles = Object.freeze({
+  pageNumber: 1,
```

Comment:
Ohh if you're a fan of SOLID, let me tell a story about why switch statements are harmful to software structure.

One upon a time...

drf-paginator is published and the software looks like:

```
Paginator
  Switch
      Style A
      Style B
```

Hannah has need for a paginator in her personal project. She uses a custom flavor of DRF and needs to use `Style H`. She starts writing the new pagination style and is soon disappointed, she needed to edit our source code to make her customization.

The design is bad—where did it go wrong?

`Paginator` calls `Style A`, the dependency points from `Paginator > Style A`. The switch statement has destroyed independent deployability. A new style is not able to be written without forking and editing the lib.

Hannah, who hates switch statements, refactors.  Opens a pull for:
```
Paginator

Style A
  Paginator

Style B
  Paginator
```

The dependencies now point up, `Style A > Paginator` (DI). Paginator is now open for extension without needing to edit (Open/closed). In her project, she's able to include `drf-paginator.js` and write separately `style-h.js`, independent deployability has been restored.

The pull is gladly accepted since it is a much better design.

Final note: switch statements are not always bad, they're often used when writing MAIN. Main calls the Application, all dependencies point the right direction and the switch statement is okay.